package.json

```diff
@@ -8,7 +8,7 @@
     "dev:integrated": "npm run devenv:build && concurrently npm:dev npm:devenv:deploy",
     "test": "cross-env NODE_ENV=development jest --config ./jest.config.js --coverage --runInBand",
     "test:watch": "cross-env NODE_ENV=development jest --config ./jest.config.js --watch",
-    "test:integration": "npm run devenv:deploy -- -d && cross-env NODE_ENV=development jest --config ./jest.config.js --coverage --no-cache --runInBand; npm run devenv:stop",
+    "test:integration": "npm run devenv:deploy -- -d && cross-env NODE_ENV=development jest --config ./jest.config.js --coverage --no-cache --runInBand && npm run devenv:stop",
```

Comment:
This one has to be `;` to function properly on *nix (cleanup should happen regardless of test exit code). There's probably some npx script to make it cross platform but too much work for now lol